app/views/admin/edition_images/crop.html.erb

```diff
@@ -0,0 +1,30 @@
+<% content_for :context, @edition.title %>
+<% content_for :page_title, "Preview image" %>
+<% content_for :title, "Preview image" %>
```

Comment:
Should the page title be "**Crop image**"?

```suggestion
<% content_for :page_title, "Crop image" %>
<% content_for :title, "Crop image" %>
```

Reply:
Double checked this with Nikin and he agreed